### ISSUE TITLE
Project Status?

### ISSUE THREAD

**Issue by novacrazy** [Ogeon/palette#71]
I see that Palette has had a lot of work done to it over the last year since the last published version, some of it not being backwards compatible.

Are there any estimates for how long until the next published version? Or perhaps publish a beta version?

I've used the master branch in my own projects without any serious issue, however crates with git deps cannot be published, so I'd be forced to work with the old published version of Palette or instruct users to use a Cargo `[replace]` section.

**Comment by Ogeon** [comment]
It has been in the back burner for about a year, since I got my current job (thus limiting my free time + energy), but I'm thinking about coming back to it soon. I would just like to make a release of Rustful first. It should not be too far into the future, but I have been visiting and been visited by family for about three weeks, so I haven't had any time at all.

All in all, I need to prioritize, but I'm happy to see that there is interest in palette and it's possible that the Rust user base may benefit more form it than from Rustful (it did never get any traction). I'll still be distracted by family until Tuesday, but then I should be able to get back to answering messages and writing code.

I'll try to make some decisions in the next few days.

**Comment by novacrazy** [comment]
What all do you have planned? I might be able to help out some in an effort to merge palette and color-rs into a definitive all-purpose color library for Rust.

**Comment by Ogeon** [comment]
The short term plan is to empty the list of stagnant PRs (either merging or
rejecting) and see if it's in good shape for a release. The long term plan
is yet to be determined, but we could aim towards some kind of merge. I
would suggest collecting some use cases (maybe a list in a tracking issue)
to get a better overview of what to include.

**Comment by Ogeon** [comment]
I have been thinking a bit more since yesterday. I'm going to look through the currently open issues, label them a bit, and decide if some of them can be closed. Then I'll see what's left to do on #60 and try to merge it. I'm going to do some gate keeping when it comes to those lengthy discussions and cut things short. Looking back, it's obvious that @sidred and I had slightly different ambitions and goals, so it was not always easy to find a common ground. From now on I'm going to declare myself The Product Owner and dictate things a bit more than before, just to avoid getting stuck again. (You are still welcome to join again, @sidred, if you are still interested)

If everything looks good and nothing important is missing, I'm going to make a release to get things out there. I can't give an ETA, since it depends on too many variables, but I'll try not to include anything more than what is already in progress.

As for the suggested Palette/color-rs merge, I think you know better than me what the shortcomings of Palette are, when it comes to GPU interoperability, so it would be great if you could either open another issue with a simple list of feature you would like to add/change, or make one for each feature if you feel like you have it nicely chunked up.

I'll let this issue linger for some more time, in case people want to know what's going on.

**Comment by Ogeon** [comment]
See the [0.3.0](https://github.com/Ogeon/palette/milestone/3) milestone for some kind of progress indication.

**Comment by Ogeon** [comment]
I think it's time for a small status update. I have started with the work on #60 and I think there's not much more left to do there. I hope I can finish it within the coming few days. Besides that, I did some issue gate keeping and assigned them to a release milestone. That's going to close a couple of issues, and then there's not much left before a release. Only the spectrum and some stuff that requires a bit of investigation.
